### PR TITLE
tsdb-labels: add tool for printing label values; rename tsdb-index

### PR DESCRIPTION
**Pull request by dimitarvdimitrov**
Added a new tool `tsdb-labels` to read the label index table and label index sections of the index. It prints each label name, the number of label values and the label values themselves.

For example

```
AlertRules 2 [false true]
ClusterName 1 [etcd]
Domain 2 [API Controller]
Event 4 [CREATE LIST READ UPDATE]
access_mode 1 [ReadWriteOnce]
acl_operation 1 [OBJECT_ACCESS_REQUIRED_OBJECT_ACL]
```

To disambiguate, tsdb-index was renamed to tsdb-series

### CODE REVIEW

File: tools/tsdb-index/main.go [hunk 1]
```diff
@@ -7,8 +7,10 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"path/filepath"
 	"time"
 
+	"github.com/efficientgo/core/errors"
```
Comment (pstibrany) [inline tools/tsdb-index/main.go:13]: Wrong import?

**Review by pstibrany — COMMENTED** [review]
Would it make sense to have this as a separate tool? Something like `tsdb-print-labels`. We don't need to put everything into `tsdb-index` (we can rename it to `tsdb-series`, as that's what it prints)

File: tools/tsdb-index/main.go [hunk 2]
```diff
@@ -40,6 +45,17 @@ func main() {
...
+	if *showLabelValues {
+		for _, blockDir := range args {
+			err = printLabelValues(ctx, blockDir)
+			if err != nil {
+				fmt.Fprintln(os.Stderr, err.Error())
+				os.Exit(1)
+			}
+		}
+		return
+	}
```
Comment (pstibrany) [inline tools/tsdb-index/main.go:58]: This is not extension of existing functionality, but completely different one ;)

Comment (dimitarvdimitrov) [reply tools/tsdb-index/main.go:58]: i think the name `tsdb-index` mislead me. I will rename the existing tool, update docs, and move the new code into `tsdb-labels`

File: CHANGELOG.md [hunk 3]
```diff
@@ -70,6 +70,7 @@
 ### Tools
 
 * [ENHANCEMENT] trafficdump: Trafficdump can now parse OTEL requests. Entire request is dumped to output, there's no filtering of fields or matching of series done. #6108
+* [ENHANCEMENT] tsdb-index: print label values via `-show-label-values`. #6317
```
Comment (pstibrany) [inline CHANGELOG.md:73]: Can we fix changelog entry?

**Review by pstibrany — APPROVED** [review]
thanks!

**Pull request merged**